ESP_SHT3x/ESP_SHT3x.ino

```diff
@@ -0,0 +1,232 @@
+int humidPin = 2;
+int tempPin = 14;
+
+// GPIO where the DS18B20 is connected to
+const int oneWireBus = 0;     
+
+// Setup a oneWire instance to communicate with any OneWire devices
+OneWire oneWire(oneWireBus);
+
+// Pass our oneWire reference to Dallas Temperature sensor 
+DallasTemperature sensors(&oneWire);
+
+// the temperature (Cecius) and humidity (%) threshholds to trigger relay
+float tempLimit = 22.78;
+float humidLimit = 97.0;
+
+// variables to influence clock controll behavior
+// records time of last change in pin voltage
+int humLastTime = 0;
+int temLastTime = 0;
+// {timeOn, timeOff} in seconds
+int humTimeDuration[2] = {10, 20};
+int temTimeDuration[2] = {5, 15};
+ 
+void setup() {
+  Serial.begin(115200);
+  // Enables DS18B20
+  sensors.begin();
+
+  // Sets the digital pins as output
+  pinMode(humidPin, OUTPUT);
+  pinMode(tempPin, OUTPUT);
+
+  // Initializes current epoch time.
+  timeClient.begin();
+  currentTime = timeClient.getEpochTime();
+
+  OTAinit();
+}
+
+// Initializes OTA functionality for the 8266
+void OTAinit() {
+  WiFi.mode(WIFI_STA);
+  WiFi.begin(ssid, password);
+  while (WiFi.waitForConnectResult() != WL_CONNECTED) {
+    Serial.println("Connection Failed! Rebooting...");
+    delay(5000);
+    ESP.restart();
+  }
+
+  ArduinoOTA.onStart([]() {
+    String type;
+    if (ArduinoOTA.getCommand() == U_FLASH) {
+      type = "sketch";
+    } else { // U_SPIFFS
+      type = "filesystem";
+    }
+
+    // NOTE: if updating SPIFFS this would be the place to unmount SPIFFS using SPIFFS.end()
+    Serial.println("Start updating " + type);
+  });
+  ArduinoOTA.onEnd([]() {
+    Serial.println("\nEnd");
+  });
+  ArduinoOTA.onProgress([](unsigned int progress, unsigned int total) {
+    Serial.printf("Progress: %u%%\r", (progress / (total / 100)));
+  });
+  ArduinoOTA.onError([](ota_error_t error) {
+    Serial.printf("Error[%u]: ", error);
+    switch (error) {
+      case OTA_AUTH_ERROR:
+        Serial.println("Auth Failed");
+        break;
+      case OTA_BEGIN_ERROR:
+        Serial.println("Begin Failed");
+        break;
+      case OTA_CONNECT_ERROR:
+        Serial.println("Connect Failed");
+        break;
+      case OTA_RECEIVE_ERROR:
+        Serial.println("Receive Failed");
+        break;
+      case OTA_END_ERROR:
+        Serial.println("End Failed");
+        break;
+    }
+  });
+  ArduinoOTA.begin();
+  Serial.println("Ready");
+  Serial.print("IP address: ");
+  Serial.println(WiFi.localIP());
+}
+
+// Outputs measured temperature and humidity to serial monitor
+void printShtMeasurments(float temp, float hum) {
+   if (! isnan(temp)) {
+    Serial.print("Temp *C = "); Serial.println(temp);
+    postProtocol("T: " + (String) temp);
+  } else { 
+    Serial.println("Failed to read temperature");
+  }
+ 
+  if (! isnan(hum)) {
+    Serial.print("Hum. % = "); Serial.println(hum);
+    postProtocol("H: " + (String) hum);
+  } else { 
+    Serial.println("Failed to read humidity");
+  }
+}
```

Comment:
Seems like this is sending temperature and humidity in two separate POST requests. They should be sent in a single request in JSON format.